Formula/zeromq.rb

```diff
@@ -32,9 +33,11 @@ def install
 
     ENV["XML_CATALOG_FILES"] = "#{etc}/xml/catalog"
 
-    ENV["LIBUNWIND_LIBS"] = "-framework System"
-    sdk = MacOS::CLT.installed? ? "" : MacOS.sdk_path
-    ENV["LIBUNWIND_CFLAGS"] = "-I#{sdk}/usr/include"
+    # Disable libunwind support due to pkg-config problem
+    # https://github.com/Homebrew/homebrew-core/pull/35940#issuecomment-454177261
+    # ENV["LIBUNWIND_LIBS"] = "-framework System"
+    # sdk = MacOS::CLT.installed? ? "" : MacOS.sdk_path
+    # ENV["LIBUNWIND_CFLAGS"] = "-I#{sdk}/usr/include"
```

Comment:
I think it's probably worth deleting rather than commenting out the code (that's what the Git history is for) and leaving the link to the PR comment.

Reply:
I did so during `brew pull --bottle`